### PR TITLE
ginkgo skip jhipster-online

### PR DESCRIPTION
**Pull request by maximilianoPizarro**
# Description of Changes

Only add --skip="stack: jhipster-online version: 2.23.0 starter: jhipster-online" from file tests/check_odov3.sh. Because fail in the step "with odo v3".

Example error in the last job:

https://github.com/devfile/registry/actions/runs/12200513685/job/34042081550

Summarizing 1 Failure:
[FAIL] test starter projects from devfile stacks [It] stack: jhipster-online version: 2.33.0 no_starter
/home/runner/work/registry/registry/tests/odov3/odo_test.go:192

### CODE REVIEW

**Review by thepetk — COMMENTED** [review]
Looks good to me, I'm just checking why the odo 3 check fails. I'm sure is not related with the current PR. I'll give it a bit of time till I re-run it if it fails again. I see the nightly run didn't exit for the same reason.

**Review by Jdubrick — COMMENTED** [review]
/ok-to-test

I think the failure in CI is related to the mirror url, it might just be down currently

**Comment by thepetk** [comment]
> /ok-to-test
> 
> I think the failure in CI is related to the mirror url, it might just be down currently

yeah I suspect so too. I guess let's give it some time

**Comment by michael-valdron** [comment]
Bug reported failing check: devfile/api#1656

**Comment by michael-valdron** [comment]
@maximilianoPizarro Suggest a rebase, if the updated Red Hat Konflux check fails we can bypass it for this PR.

File: tests/check_odov3.sh [hunk 1]
```diff
@@ -49,6 +49,7 @@ ginkgo run --procs 2 \
   --skip="stack: java-vertx version: 1.4.0 starter: vertx-istio-security-booster" \
   --skip="stack: java-vertx version: 1.4.0 starter: vertx-messaging-work-queue-booster" \
   --skip="stack: java-websphereliberty-gradle version: 0.4.0 starter: rest" \
+  --skip="stack: jhipster-online version: 2.23.0 starter: jhipster-online" \
```
Comment (michael-valdron) [inline tests/check_odov3.sh:52]: Your stack version might be a typo:
```suggestion
  --skip="stack: jhipster-online version: 2.33.0 starter: jhipster-online" \
```

**Review by michael-valdron — CHANGES_REQUESTED** [review]

File: tests/check_odov3.sh (hunk 1 above)
Comment (maximilianoPizarro) [reply tests/check_odov3.sh]: Thanks Michael, I send the change.

**Review by michael-valdron — APPROVED** [review]
/lgtm

**Comment by maximilianoPizarro** [comment]
Hi Michael again.
I see there are other projects that are affected. 

```bash
Container test exited with code 1, reason Error
---
led
+ for test in '"${FAILED_TESTS[@]}"'
+ echo 'name: nodejs-nuxtjs version: 1.0.3 error: create failed'
name: nodejs-nuxtjs version: 1.0.3 error: create failed
+ for test in '"${FAILED_TESTS[@]}"'
+ echo 'name: nodejs-nuxtjs version: 1.1.0 error: create failed'
name: nodejs-nuxtjs version: 1.1.0 error: create failed
+ for test in '"${FAILED_TESTS[@]}"'
+ echo 'name: nodejs-react version: 2.0.2 error: unable to get working url'
name: nodejs-react version: 2.0.2 error: unable to get working url
+ for test in '"${FAILED_TESTS[@]}"'
+ echo 'name: nodejs-react version: 2.1.0 error: unable to get working url'
name: nodejs-react version: 2.1.0 error: unable to get working url
+ for test in '"${FAILED_TESTS[@]}"'
+ echo 'name: nodejs-svelte version: 1.0.3 error: unable to get working url'
name: nodejs-svelte version: 1.0.3 error: unable to get working url
+ for test in '"${FAILED_TESTS[@]}"'
+ echo 'name: nodejs-svelte version: 1.1.0 error: unable to get working url'
name: nodejs-svelte version: 1.1.0 error: unable to get working url
+ for test in '"${FAILED_TESTS[@]}"'
+ echo 'name: nodejs-vue version: 1.0.2 error: unable to get working url'
name: nodejs-vue version: 1.0.2 error: unable to get working url
+ for test in '"${FAILED_TESTS[@]}"'
+ echo 'name: nodejs-vue version: 1.1.0 error: unable to get working url'
name: nodejs-vue version: 1.1.0 error: unable to get working url
+ for test in '"${FAILED_TESTS[@]}"'
+ echo 'name: php-laravel version: 1.0.1 error: create failed'
name: php-laravel version: 1.0.1 error: create failed
+ for test in '"${FAILED_TESTS[@]}"'
+ echo 'name: python version: 2.1.0 error: create failed'
name: python version: 2.1.0 error: create failed
+ exit 1
```

Would you like me to add them to the ginkgo skip list in this PR?

**Comment by michael-valdron** [comment]
> Hi @michael-valdron again. I see there are other projects that are affected.
> 
> ```shell
> Container test exited with code 1, reason Error
> ---
> led
> + for test in '"${FAILED_TESTS[@]}"'
> + echo 'name: nodejs-nuxtjs version: 1.0.3 error: create failed'
> name: nodejs-nuxtjs version: 1.0.3 error: create failed
> + for test in '"${FAILED_TESTS[@]}"'
> + echo 'name: nodejs-nuxtjs version: 1.1.0 error: create failed'
> name: nodejs-nuxtjs version: 1.1.0 error: create failed
> + for test in '"${FAILED_TESTS[@]}"'
> + echo 'name: nodejs-react version: 2.0.2 error: unable to get working url'
> name: nodejs-react version: 2.0.2 error: unable to get working url
> + for test in '"${FAILED_TESTS[@]}"'
> + echo 'name: nodejs-react version: 2.1.0 error: unable to get working url'
> name: nodejs-react version: 2.1.0 error: unable to get working url
> + for test in '"${FAILED_TESTS[@]}"'
> + echo 'name: nodejs-svelte version: 1.0.3 error: unable to get working url'
> name: nodejs-svelte version: 1.0.3 error: unable to get working url
> + for test in '"${FAILED_TESTS[@]}"'
> + echo 'name: nodejs-svelte version: 1.1.0 error: unable to get working url'
> name: nodejs-svelte version: 1.1.0 error: unable to get working url
> + for test in '"${FAILED_TESTS[@]}"'
> + echo 'name: nodejs-vue version: 1.0.2 error: unable to get working url'
> name: nodejs-vue version: 1.0.2 error: unable to get working url
> + for test in '"${FAILED_TESTS[@]}"'
> + echo 'name: nodejs-vue version: 1.1.0 error: unable to get working url'
> name: nodejs-vue version: 1.1.0 error: unable to get working url
> + for test in '"${FAILED_TESTS[@]}"'
> + echo 'name: php-laravel version: 1.0.1 error: create failed'
> name: php-laravel version: 1.0.1 error: create failed
> + for test in '"${FAILED_TESTS[@]}"'
> + echo 'name: python version: 2.1.0 error: create failed'
> name: python version: 2.1.0 error: create failed
> + exit 1
> ```
> 
> Would you like me to add them to the ginkgo skip list in this PR?

@maximilianoPizarro Thanks for offering but not necessary, these failures are caused by an issue with our testing environment: https://github.com/devfile/api/issues/1663

Also another issue regarding other stacks using the 2.2.2 spec by default but not being exempt from the odo v2 testing that our integration testing uses.

For your PR, I can run our integration testing locally for you tomorrow and bypass the failing check if everything passes as nothing I am seeing is being cause by your changes.

**Review by michael-valdron — APPROVED** [review]
/lgtm

@maximilianoPizarro I was able to run your change successfully on my side: [test_log.txt](https://github.com/user-attachments/files/18202371/test_log.txt)

All but the failures due to devfile/api#1664:
```
The following tests failed:
name: java-springboot version: 1.2.0 error: create failed
name: java-springboot version: 1.3.0 error: create failed
name: java-wildfly version: 1.1.0 error: create failed
name: nodejs version: 2.1.1 error: create failed
name: nodejs version: 2.2.0 error: create failed
name: nodejs-angular version: 2.0.2 error: create failed
name: nodejs-angular version: 2.1.0 error: create failed
name: nodejs-nextjs version: 1.0.3 error: create failed
name: nodejs-nextjs version: 1.1.0 error: create failed
name: nodejs-nuxtjs version: 1.0.3 error: create failed
name: nodejs-nuxtjs version: 1.1.0 error: create failed
name: php-laravel version: 1.0.1 error: create failed
name: python version: 2.1.0 error: create failed
name: java-maven version: 1.2.0 error: create failed
name: java-maven version: 1.3.0 error: create failed
name: java-vertx version: 1.2.0 error: create failed
name: java-vertx version: 1.3.0 error: create failed
```

Since these are unrelated to your change and this stack is not on this list I will bypass the protection to merge your PR.

@thepetk Does everything look good from your side?

**Review by thepetk — APPROVED** [review]

**Comment by thepetk** [comment]
> /lgtm
> 
> @maximilianoPizarro I was able to run your change successfully on my side: [test_log.txt](https://github.com/user-attachments/files/18202371/test_log.txt)
> 
> All but the failures due to devfile/api#1664:
> ```
> The following tests failed:
> name: java-springboot version: 1.2.0 error: create failed
> name: java-springboot version: 1.3.0 error: create failed
> name: java-wildfly version: 1.1.0 error: create failed
> name: nodejs version: 2.1.1 error: create failed
> name: nodejs version: 2.2.0 error: create failed
> name: nodejs-angular version: 2.0.2 error: create failed
> name: nodejs-angular version: 2.1.0 error: create failed
> name: nodejs-nextjs version: 1.0.3 error: create failed
> name: nodejs-nextjs version: 1.1.0 error: create failed
> name: nodejs-nuxtjs version: 1.0.3 error: create failed
> name: nodejs-nuxtjs version: 1.1.0 error: create failed
> name: php-laravel version: 1.0.1 error: create failed
> name: python version: 2.1.0 error: create failed
> name: java-maven version: 1.2.0 error: create failed
> name: java-maven version: 1.3.0 error: create failed
> name: java-vertx version: 1.2.0 error: create failed
> name: java-vertx version: 1.3.0 error: create failed
> ```
> 
> Since these are unrelated to your change and this stack is not on this list I will bypass the protection to merge your PR.
> 
> @thepetk Does everything look good from your side?

Yeah tbh I don't see any issues merging this PR

**Pull request merged**